src/main/java/duke/component/Storage.java

```diff
@@ -0,0 +1,76 @@
+package duke.component;
+
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+import java.io.*;
```

Comment:
Shouldn't the classes imported from java.io be spelt out rather than lumping them together into a wildcard?

Reply:
Thanks for the input! I have refactored accordingly.